e2e-tests/visual-regression/cypress/plugins/index.js

```diff
@@ -17,7 +17,7 @@ module.exports = (on, config) => {
   on("before:browser:launch", (browser = {}, launchOptions) => {
     if (browser.family === "chromium" || browser.family === "chrome") {
       // Make retina screens run at 1x density so they match the versions in CI
-      launchOptions.push("--force-device-scale-factor=1")
+      launchOptions.args.push("--force-device-scale-factor=1")
```

Comment:
This was an API change from Cypress: https://docs.cypress.io/guides/references/migration-guide#Plugin-Event-before-browser-launch